spec/rubocop/cop/lint/no_return_in_begin_end_blocks_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::NoReturnInBeginEndBlocks do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  def return_inside_or_assignment_block
```

Comment:
Doesn't appear to be used

Reply:
Yes, you're right. I also think this and the other method are not necessary anymore. I'll remove them from the test as the shared examples are covering all the situations now. 👍